config/initializers/carrier_wave.rb

```diff
@@ -0,0 +1,18 @@
+CarrierWave.configure do |config|
+  if Rails.env.staging? || Rails.env.production?
+    config.storage    = :aws
+    config.aws_bucket = ENV.fetch('S3_BUCKET_NAME')
+    config.aws_acl    = :'public-read'
+    config.asset_host = 'http://example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.